routes/api/contacts.js

```diff
@@ -84,13 +80,62 @@ router.post("/", async (req, res) => {
   }
 });
 
-// DELETE /api/contacts/:id
-router.delete("/:id", async (req, res) => {
-  const deletedContact = await contactsModel.removeContact(req.params.id);
+// DELETE /api/contacts/:contactId
+router.delete("/:contactId", async (req, res) => {
+  const deletedContact = await contactsModel.removeContact(
+    req.params.contactId
+  );
   if (!deletedContact) {
     return res.status(404).json({ message: "Not found" });
   }
   res.json({ message: "contact deleted" });
 });
 
+// Funkcja aktualizująca status kontaktu w bazie danych
+async function updateStatusContact(contactId, favorite) {
+  try {
+    console.log("Updating contact with ID:", contactId);
+    console.log("New favorite value:", favorite);
+    const updatedContact = await mongoose
+      .model("Contact")
+      .findByIdAndUpdate(contactId, { favorite: favorite }, { new: true });
```

Comment:
```suggestion
    const updatedContact = await User
      .findByIdAndUpdate(contactId, { favorite: favorite }, { new: true });
```
Tu trzeba użyć instancji modelu exportowanego z contactsSchema.js